doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1301 @@
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v3.0.0`_ for the changes required or recommended when migrating your application from |NCS| v2.9.0 to |NCS| v3.0.0.
+
+.. _ncs_release_notes_300_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE, OS, and tool support
+=========================
+
+* Updated:
+
+  * The required `SEGGER J-Link`_ version to v8.18.
+  * The :ref:`installation` page with the following:
+
+    * The :ref:`installing_vsc` section with a list valid for both development environments.
+      The list now includes nRF Util as an additional requirement for :ref:`west runner <programming_selecting_runner>`  for the |nRFVSC|, and the Windows-only requirement to install SEGGER USB Driver for J-Link for both development environments.
+    * The command-line instructions now use the ``nrfutil sdk-manager`` command instead of the ``nrfutil toolchain-manager`` command.
+      You can read more about the new command in the `nRF Util documentation <sdk-manager command_>`_.
+
+  * Mentions of commands that use tools from the nRF Command Line Tools to use nRF Util.
+    |nrf_CLT_deprecation_note|
+
+* Removed the Toolchain Manager app section from the following pages:
+
+  * :ref:`installation`
+  * :ref:`updating`
+  * :ref:`requirements`
+
+  The app has been deprecated and no longer provides the latest toolchain and |NCS| versions for installation.
+
+Board support
+=============
+
+* Removed support for the nRF52810 Desktop Mouse board (``nrf52810dmouse/nrf52810``).
+
+Build and configuration system
+==============================
+
+* Removed support for the deprecated multi-image builds (parent-child images) functionality.
+  All |NCS| projects must now use :ref:`sysbuild`.
+  See :ref:`child_parent_to_sysbuild_migration` for an overview of differences with parent-child image and how to migrate.
+* Updated:
+
+  * The default runner for the ``west flash`` command to `nRF Util`_ instead of ``nrfjprog`` that is part of the archived `nRF Command Line Tools`_.
+    For more information, see the :ref:`build system section in the v3.0.0 migration guide <migration_3.0_recommended>` and the :ref:`programming_selecting_runner` section on the programming page.
+  * Erasing the external memory when programming a new firmware image with ``west flash`` series now always correctly honors the ``--erase`` flag (and its absence) both when using the ``nrfjprog`` and ``nrfutil`` backends.
+    Prior to this release, the ``nrjfprog`` backend would always erase only the sectors of the external flash used by the new firmware, and the ``nrfutil`` backend would always erase the whole external flash.
+  * :kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE` No longer depends on external flash (NCSIDB-1232).
+  * Static partition manager file for network core images can now be set (NCSIDB-1442).
+
```

Comment:
Please add:
```suggestion
Bootloader and DFU
=============
* Added experimental support |NSIB| for nRF54L15 SoC. On nRF54L SoCs NSIB uses KMU for authentication key storage, it supports keys revocation scheme and it can be protected using immutable-boot region SoC's hardware feature.
* Improve DFU timing performance on nRF54l SoC by applying optimal RRAMC buffering.
```

Reply:
Added